### DISCUSSION TITLE
n8n LXC installment giving 401 error when setting up user.

### DISCUSSION THREAD

**Discussion by roskul82** [tteck/Proxmox#2638] [Help Needed] [answered]
Had n8n up and running for a while with no problems.
Had one workflow that ran 1 time a week. 
Suddenly it gave me error that it was running out of space. Tried fixing it, but broke the install.

Oh well, decided to just spin up a new one and set it up again.

But now it just gives me this error everytime I'm trying to make a new user.

> Problem setting up owner
> 
> Request failed with status code 401 

Looking at the n8n forums it seems to be related to this:

> Are you accessing your n8n instance over HTTP or HTTPS? If you are using HTTP you may see this issue and it would be worth trying to set the N8N_SECURE_COOKIE env option to false and seeing if that helps.

[https://community.n8n.io/t/problem-setting-up-owner-request-failed-with-status-code-401/42308](url)

I'm a quite fresh (noob) in this. Is there a way for me to fix this on the current install? Or is this something that needs to be done via the script?

**Comment by tteck** [comment, marked as answer]
Environment option added https://github.com/tteck/Proxmox/commit/e5010e747b8fea9b9b94024050bd2ab635ab2a20
Storage added https://github.com/tteck/Proxmox/commit/8e71d636efbe44d8a7fedb017762fdf4aec485fa

**Reply by roskul82** [reply]
Works as a charm now.

Thanks alot!
